python/pyspark/resource/taskresourcerequests.py

```diff
@@ -0,0 +1,52 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from pyspark.resource.taskresourcerequest import TaskResourceRequest
+
+
+class TaskResourceRequests(object):
+
+    """
+    .. note:: Evolving
+
+    A set of task resource requests. This is used in conjuntion with the
+    ResourceProfileBuilder to programmatically specify the resources needed for
+    an RDD that will be applied at the stage level.
+    """
+
+    def __init__(self):
+        """Create a new TaskResourceRequests that wraps the underlying JVM object."""
+        from pyspark import SparkContext
+        self._javaTaskResourceRequests \
```

Comment:
nit: `_javaTaskResourceRequests` -> `_java_task_resource_requests`. When the name is private, we don't have to follow `camelCase`. Surprisingly, such naming style kind of follows PEP 8 - there's a similar case in `threading` in Python standard library.